pkg/mocks/context.go

```diff
@@ -0,0 +1,144 @@
+package mocks
```

Comment:
Do we need this? See:https://github.com/dcos/dcos-core-cli/blob/55b3085ed15a1b26df8f0fbce58c15958c2541cc/vendor/github.com/dcos/dcos-cli/pkg/mock/mock.go#L82-L92